openapi/platform/platform.yaml

```
@@ -3068,6 +3068,12 @@ components:
                       - monthly
                       - yearly
                     description: The billing frequency
+                  isPackage:
+                    type: boolean
+                    description: Indicates if this lineitem is a package instead of a product or an addon.
+                  containedInPackage:
+                    type: string
+                    description: Holds the SKU of the package that this lineitem is part of. As products inside of a package does not currently have pricing you should look to the lineitem of the package SKU to find the price.
```

minor wording suggestion
```suggestion
                    description: Holds the SKU of the package that this lineitem is part of. Products inside of a package do not have pricing so refer to the lineitem of the package SKU to find the price.
```